internal/gitstream/assign.go

```diff
@@ -103,12 +103,17 @@ func (a *Assign) assignIssues(ctx context.Context) error {
 
 			logger.Info("Assigning issue")
 
+			var userLogin string
 			user, err := a.UserHelper.GetUser(ctx, upstreamCommit)
 			if err != nil {
-				return fmt.Errorf("could not get the upstream commit author for downstream issue %d: %v", *issue.Number, err)
+				logger.Info("WARNING: could not get the upstream commit author for downstream issue, picking a random assignee",
+					"issue", *issue.Number)
+				userLogin = "notanowner"
```

Comment:
https://github.com/qbarrand/gitstream/pull/29#discussion_r1087826914

Reply:
I don't think this is a good idea. IMO it would be better to have a `getRandomAssignee` method to the `Owners` type - that would make the whole thing more explicit.

Reply:
Fixed. PTAL.